R/explanation.R

```diff
@@ -92,9 +92,7 @@
 #' explain4 <- explain(x_test, explainer, approach = approach, prediction_zero = p, n_samples = 1e2)
 #'
 #' # Plot the results
-#' \dontrun{
```

Comment:
The removal of this comes from a comment @JensWahl got from CRAN on another package: 

"\dontrun{} should only be used if the example really cannot be executed
(e.g. because of missing additional software, missing API keys, ...) by
the user. That's why wrapping examples in \dontrun{} adds the comment
("# Not run:") as a warning for the user.
Does not seem necessary.
Please unwrap the examples if they are executable in < 5 sec, or replace
\dontrun{} with \donttest{}."

I suspect we would get the same reply for our usage, so I just removed it before submitting the first time.

Reply:
Sounds good!